prog/vm/github_runner.rb

```diff
@@ -154,6 +152,7 @@ def before_run
     picked_vm = pick_vm
     github_runner.update(vm_id: picked_vm.id)
     picked_vm.update(name: github_runner.ubid.to_s)
+    github_runner.reload.log_duration("runner allocated", Time.now - github_runner.created_at)
```

Comment:
do we have to reload here?

Reply:
We updated `vm_id` just two lines ago, but it returns `nil` without reloading. I could attempt to use the `github_runner.add_vm(picked_vm)` method from Sequel. This might update the `.vm` association.

Reply:
`github_runner.add_vm(picked_vm)` might have other unintended side effects (need to check the sequel code)

Reply:
`github_runner.add_vm(picked_vm)` might have other unintended side effects (need to check sequel code)

Reply:
Since we don't have example of `add_*` in our codebase, I'll leave it as is.